core/src/main/java/com/novoda/noplayer/internal/exoplayer/ExoPlayerFacade.java

```diff
@@ -114,7 +107,8 @@ void loadVideo(SurfaceHolder surfaceHolder,
                    ExoPlayerForwarder forwarder,
                    MediaCodecSelector mediaCodecSelector) {
         this.contentType = contentType;
-        exoPlayer = exoPlayerCreator.create(drmSessionCreator, forwarder.drmSessionEventListener(), mediaCodecSelector);
+        trackSelector = trackSelectorCreator.create();
```

Comment:
@zegnus this line is where we now creation the track selector, rather than having all the track selectors precreated